api/external/cfgmgmt/response/suggestions.proto

```diff
@@ -4,6 +4,8 @@ package chef.automate.api.cfgmgmt.response;
 option go_package = "github.com/chef/automate/api/external/cfgmgmt/response";
 
 message Suggestion {
+  // Text to search on for the suggestions.
   string text = 1;
+  // Score of the result from ElasticSearch.
```

Comment:
Do we want to mention that a larger score is more heavily weighted?

Reply:
hmm. i'm not sure it's super important information to say that (and also kinda obvious?) -- not sure how useful it is to the user -- @kagarmoe thoughts?